### PR TITLE
Locallab blur cleanup and small speedup

### PR DESCRIPTION
**Pull request by heckflosse**

### CODE REVIEW

File: rtengine/CMakeLists.txt [hunk 1]
```diff
@@ -79,6 +79,7 @@ set(RTENGINESOURCEFILES
     impulse_denoise.cc
     init.cc
     iplab2rgb.cc
+	iplocallab.cc
```
Comment (Beep6581) [inline rtengine/CMakeLists.txt:82]: @heckflosse indentation seems off (when viewed in GitHub)

File: rtdata/languages/default [hunk 2]
```diff
@@ -692,21 +692,37 @@ HISTORY_MSG_441;Retinex - Gain transmission
...
-HISTORY_MSG_465;PS - Blur radius
-HISTORY_MSG_468;PS - Fill holes
-HISTORY_MSG_469;PS - Median
-HISTORY_MSG_471;PS - Motion correction
-HISTORY_MSG_472;PS - Smooth transitions
-HISTORY_MSG_473;PS - Use LMMSE
-HISTORY_MSG_474;PS - Equalize
-HISTORY_MSG_475;PS - Equalize channel
+HISTORY_MSG_445;Raw Sub-Image
+HISTORY_MSG_446;EvPixelShiftMotion
```
Comment (Beep6581) [inline rtdata/languages/default:696]: Are these keys HISTORY_MSG_446 - 475 used in the "Raw" tab, or by the Local tool? If the latter, why aren't they prefixed with "Local - " same as HISTORY_MSG_515 - 661?

**Comment by Beep6581** [comment]
A bit confused about all this. I haven't seen branch `locallab_burr_cleanup` before, but this PR makes it seem like this branch has been around since 2018, that it has a whopping 581 commits, and that you want to merge it into `dev`, none of which makes sense. Also unclear how this fits in with the other locallab speedup branches `newlocallab-speedup` `newlocallab-speedup2`.

**Comment by heckflosse** [comment]
@Beep6581 This was nonsense. I forgot to change the destination branch to newlocallab. Then I set it to newlocallab and forgot to close this one.

**Pull request closed without merging**